dev-requirements.txt

```diff
@@ -1,9 +1,6 @@
 -e .
 flake8<3.0
 invoke>=0.11.1,<0.12
-pytest
-pytest-cov
-pytest-django
```

Comment:
Do you think we need to remove the `pytest` packages here? Sometimes it's useful to do a quick test run on the current Python interpreter. If you still think we should remove these, then the reference to `py.test` should probably be removed from the [Contributing > Running Tests](https://django-elevate.readthedocs.io/en/latest/contributing/index.html#running-tests) docs.

Reply:
I think I'll add it back in. I've since tried to do things like tox -e py27 and found that it does not work as I had assumed.

Reply:
Okay, cool. Might consider doing the same with `flake8`, since we presumably want to encourage folks to use it. 😉